src/subdomains/analytics/application/services/staking-history.service.ts

```diff
@@ -47,6 +44,8 @@ export class StakingHistoryService {
     const transactions = (
       exportFormat === ExportType.CT
         ? this.getHistoryCT(deposits, withdrawals, rewards)
+        : exportFormat === ExportType.CHAIN_REPORT
+        ? this.getHistoryChainReport(deposits, withdrawals, rewards)
         : this.getHistoryCompact(deposits, withdrawals, rewards)
     ) as HistoryDto<T>[];
 
```

Comment:
This would be more readable in a switch case, e.g.
```ts
switch (exportFormat) {
   case ExportType.COIN_TRACKING:
      return this.getHistoryCT(deposits, withdrawals, rewards);
   case ExportType.CHAIN_REPORT:
      return this. getHistoryChainReport(deposits, withdrawals, rewards);
   case ExportType.COMPACT:
      return this.getHistoryCompact(deposits, withdrawals, rewards);
}
```